### PR TITLE
Migrate to the allmighty monorepode

### PR DESCRIPTION
**Pull request by arrdem**
image

MONOREPO IS LOVE MONPREPO IS LIFE

Just so that the version strings are always in sync. Could probably add a `Makefile` which wraps building, testing, installing & deploying both projects.

### CODE REVIEW

File: lein-kibit/src/leiningen/kibit.clj [hunk 1]
```diff
@@ -0,0 +1,28 @@
+(ns leiningen.kibit
+  (:require [leiningen.core.eval :refer [eval-in-project]]
+            [clojure.tools.namespace.find :refer [find-namespaces]])
+  (:import [java.io File]))
+
+
+(defn ^:no-project-needed kibit
+  [project & args]
+  (let [src-paths (get-in project [:kibit :source-paths] ["rules"])
+        kibit-project `{:dependencies [[jonase/kibit "0.1.3"]]
```
Comment (danielcompton) [inline lein-kibit/src/leiningen/kibit.clj]: this needs to have version update too

File: lein-kibit/project.clj [hunk 2]
```diff
@@ -0,0 +1,9 @@
+(defproject lein-kibit "0.1.4-SNAPSHOT"
+  :description "kibit lein plugin"
+  :url "https://github.com/jonase/lein-kibit"
+  :license {:name "Eclipse Public License"
+            :url  "http://www.eclipse.org/legal/epl-v10.html"}
+  :dependencies [[jonase/kibit "0.1.4-SNAPSHOT"]
```
Comment (danielcompton) [inline lein-kibit/project.clj]: and this

Comment (arrdem) [inline lein-kibit/project.clj]: So it looks like we can drop this particular dependency instance altogether. The leiningen plugin doesn't directly load any kibit code, it just drives the user's project with the matching kibit dependency. The patches below refactor the VERSION file out to a common resource tree, and load it from there (or include it in the jar) as appropriate.

File: .gitignore [hunk 3]
```diff
@@ -15,7 +16,7 @@ pom.xml.asc
 .lein-env
 .lein-repl-history
 
-# Temp Files
+# Temp Filest
```
Comment (danielcompton) [inline .gitignore]: wat

**Comment by danielcompton** [comment]
I think this tests are looking good now. Anything more to come?

**Comment by arrdem** [comment]
I'll pull your changes, make sure and squash sometime today. No need to inter my/our fumbling in the git log forever :stuck_out_tongue:

**Comment by arrdem** [comment]
I wrote new, better commits with the same diffs. This should be ready to merge. We could probably add some release automation, but this is already a step in the right direction.

**Comment by danielcompton** [comment]
[https://68.media.tumblr.com/644c298eafe77069787dcc2674c630f1/tumblr_olt4qsxPBW1txygfho1_500.jpg]

**Pull request merged**